precode.go

```diff
@@ -13,29 +15,54 @@ import (
 // сгенерированных чисел.
 func Generator(ctx context.Context, ch chan<- int64, fn func(int64)) {
 	// 1. Функция Generator
-	// ...
+	defer close(ch)
+	var n, counter int64
+	for {
+		if counter == 0 {
+			n = 1
+		} else {
+			n = (counter - 1) + 1
```

Comment:
Это выражение по сути равно. Если нужно считать с 1, то можно сделать `counter = 1`
```suggestion
			n = counter
```